File: BEmuJava/bemu/src/main/java/com/bloomberglp/blpapi/SchemaElementDefinition.java
//------------------------------------------------------------------------------
// <copyright project="BEmu_maven" file="/BEmu_maven/bemu/src/main/java/com/bloomberglp/blpapi/SchemaElementDefinition.java" company="Jordan Robinson">
//     Copyright (c) 2013 Jordan Robinson. All rights reserved.
//
//     The use of this software is governed by the Microsoft Public License
//     which is included with this distribution.
// </copyright>
//------------------------------------------------------------------------------

package com.bloomberglp.blpapi;

public class SchemaElementDefinition
{
	public static final int UNBOUNDED = -1;
	
	private SchemaElementDefinition() { }
	
	public void setUserData(Object arg0) throws Exception
	{
		throw new Exception("not implemented");
	}
	
	public Name name() throws Exception
	{
		throw new Exception("not implemented");
	}
	
	public int numAlternateNames() throws Exception
	{
		throw new Exception("not implemented");
	}
	
	public Name getAlternateName(int arg0) throws Exception
	{
		throw new Exception("not implemented");
	}
	
	public String description() throws Exception
	{
		throw new Exception("not implemented");
	}
	
	public Schema.Status status() throws Exception
	{
		throw new Exception("not implemented");
	}
	
	public SchemaTypeDefinition typeDefinition() throws Exception
	{
		throw new Exception("not implemented");
	}
	
	public int maxValues() throws Exception
	{
		throw new Exception("not implemented");
	}
	
	public ConstraintsList constraints() throws Exception
	{
		throw new Exception("not implemented");
	}
	
	public Object userData() throws Exception
	{
		throw new Exception("not implemented");
	}	
}

File: BEmu/BloombergTypes/SchemaElementDefinition.cs
//------------------------------------------------------------------------------
// <copyright project="BEmu_csh" file="BloombergTypes/SchemaElementDefinition.cs" company="Jordan Robinson">
//     Copyright (c) 2013 Jordan Robinson. All rights reserved.
//
//     The use of this software is governed by the Microsoft Public License
//     which is included with this distribution.
// </copyright>
//------------------------------------------------------------------------------

namespace Bloomberglp.Blpapi
{
    using System;
    using System.Collections.Generic;
    using System.Linq;
    using System.Text;

    public class SchemaElementDefinition
    {
        public const int UNBOUNDED = -1;

        private SchemaElementDefinition() { }

        public IEnumerable<Name> AlternateNames { get { throw new NotImplementedException(); } }
        public ConstraintsList Constraints { get { throw new NotImplementedException(); } }
        public string Description { get { throw new NotImplementedException(); } }
        public int MaxValues { get { throw new NotImplementedException(); } }
        public int MinValues { get { throw new NotImplementedException(); } }
        public Name Name { get { throw new NotImplementedException(); } }
        public int NumAlternateNames { get { throw new NotImplementedException(); } }
        public Schema.Status Status { get { throw new NotImplementedException(); } }
        public SchemaTypeDefinition TypeDefinition { get { throw new NotImplementedException(); } }
        
        public object UserData
        {
            get { throw new NotImplementedException(); }
            set { throw new NotImplementedException(); }
        }

        public Name GetAlternateName(int index)
        {
            throw new NotImplementedException();
        }
    }
}